editor.planx.uk/src/pages/layout/PublicLayout.tsx

```diff
@@ -96,7 +96,7 @@ const PublicLayout: React.FC<PropsWithChildren> = ({ children }) => {
     <StyledEngineProvider injectFirst>
       <ThemeProvider theme={teamMUITheme}>
         <Header />
-        <MainContainer id="main-content">
```

Comment:
_Question:_

I think the `Skip To Main Content` might have been broken by this change and the target for the content maybe needs to be updated: https://github.com/theopensystemslab/planx-new/blob/67d6f182634dc5c57804e9971ce290abe3525abb/editor.planx.uk/src/components/Header.tsx#L348

On dev `skip to main content` goes to the `back` button but on the pizza I think it doesn't go anywhere when it's selected. 

https://github.com/theopensystemslab/planx-new/assets/36415632/6d97fa40-a245-4113-b8df-f878029b3097

Reply:
Thanks for spotting this and testing on the Pizza - this wasn't the result I was getting was getting locally. I think that as `Node` just yields to a child it wasn't handling the new properties correctly.

This is fixed by making an explicit main component in [0f01730](https://github.com/theopensystemslab/planx-new/pull/2970/commits/0f01730b33ba909b525e74f9d7e0bc31d74ac2f0) 👍